src/cli.rs

```diff
@@ -7,12 +7,25 @@ use structopt::clap::AppSettings::{ColorAlways, ColoredHelp, DeriveDisplayOrder}
 use structopt::{clap, StructOpt};
 use syntect::highlighting::Theme as SyntaxTheme;
 use syntect::parsing::SyntaxSet;
+use unicode_segmentation::UnicodeSegmentation;
 
 use crate::bat_utils::assets::HighlightingAssets;
 use crate::bat_utils::output::PagingMode;
 use crate::git_config::{GitConfig, GitConfigEntry};
 use crate::options;
 
+pub const INLINE_SYMBOL_WIDTH_1: usize = 1;
+
+fn ensure_display_width(arg: &str) -> Result<String, String> {
```

Comment:
This could be called `ensure_inline_symbol_display_width`, and it would be nice to have a comment indicating what an "inline symbol" is.